src/controllers/google/LimitedDevicesController.ts

```diff
@@ -32,11 +34,23 @@ export class LimitedDevicesController {
     }
 
     @Get('/token-with-user')
-    async getTokenAndUserResponse(@QueryParam('device_code', { required: true }) device_code: string) {
+    async getTokenAndUserResponse(
+        @QueryParam('device_code', { required: true }) device_code: string,
+        @Res() res: e.Response,
+    ) {
         if (!device_code) {
             throw new Error(`Missing "device_code"`);
         }
-        const tokenResponse = await this.client.getTokenResponse(device_code);
-        return this.client.getUserInfo(tokenResponse);
+        try {
+            const tokenResponse = await this.client.getTokenResponse(device_code);
+            return this.client.getUserInfo(tokenResponse);
+        } catch (e) {
+            if (isAxiosError(e) && e.response?.status === 428) {
+                res.status(202);
+                return e.response.data;
+            }
+
+            throw e;
+        }
```

Comment:
that's because the 428 error is the way Google says "Not yet, try again later". I do not like the fact that they return a 428 so here i interpret the 428 as a 202. This is all correct.